### DISCUSSION TITLE
Dependabot updating a Actions Minor and Patch Version

### DISCUSSION THREAD

**Discussion by RainerZufall187** [community/community#49734] [Code Security] [answered]
### Select Topic Area

Question

### Body

So I want Dependabot to really update my action workflow steps to its very latest version and not to their latest major.

For example dependabot created a pr "Bump actions/setup-python from 3 to 4" but I want it to be "Bump actions/setup-python from 3 to 4.5.0".

Is this possible?

**Comment by jeffwidman** [comment, marked as answer]
No, it respects the specificity of the previous tag. This is works-as-designed because most users pick what level of pin they want.

So in your case you'd want to pin to `actions/setup-python@3.x.0` probably, and then it'll bump it up to `4.5.0`.
